int32[12] loaded

